main.js

```diff
@@ -0,0 +1,137 @@
+'use strict';

```

Comment:
I know it was a bit vague this time around since I gave the links to the MDN repo, but in the future, it would be best to include the reference source for the code segment.